### PR TITLE
Create Security.md file

### PR DESCRIPTION
**Pull request by mahrufiqbalHO**
## Purpose of pull request
Create security.md file as per the best practices 

## How to test
read the security.md context file

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- demand money to disclose a vulnerability
+
+Only submit reports about exploitable vulnerabilities through HackerOne.
+
+## Bug bounty
+Unfortunately, HomeOffice doesn't offer a paid bug bounty programme. HomeOffice will make efforts to show appreciation to people who take the time and effort to disclose vulnerabilities responsibly.
+
+# Code of Conduct
+
+HomeOffice have a contributors code of conduct, which you can find here: [CODE_OF_CONDUCT.md](https://github.com/HO-CTO/sre-monitoring-as-code/blob/main/CODE_OF_CONDUCT.md)
```
Comment (samiwelthomasHO) [inline SECURITY.md:49]:
```suggestion
UKHomeOffice have a contributors code of conduct, which you can find here: [CODE_OF_CONDUCT.md](https://github.com/HO-CTO/sre-monitoring-as-code/blob/main/CODE_OF_CONDUCT.md)
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+---
+
+#### Further reading and inspiration about responsible disclosure and `SECURITY.md`
+- <https://www.gov.uk/help/report-vulnerability>
+- <https://mojdigital.blog.gov.uk/vulnerability-disclosure-policy/>
+- <https://www.ncsc.gov.uk/information/vulnerability-reporting>
+- <https://github.com/Trewaters/security-README>
+
+[<disclosure-email-address>]: mailto:<disclosure-email-address>
```
Comment (samiwelthomasHO) [inline SECURITY.md:59]: Add SRE Team email here

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+[CODE_OF_CONDUCT.md]: https://github.com//.github/blob/master/CODE_OF_CONDUCT.md
```
Comment (samiwelthomasHO) [inline SECURITY.md:60]: Should link to our code of conduct.

File: security.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Policy: https://www.gov.uk/help/report-vulnerability
+
+Contact: https://hackerone.com/<>/embedded_submissions/new
```
Comment (samiwelthomasHO) [inline security.txt:3]: Remove

File: security.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+Contact: https://www.gov.uk/contact/govuk
```
Comment (samiwelthomasHO) [inline security.txt:4]: SRE Team email?

File: security.txt [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+Hiring: https://www.civilservicejobs.service.gov.uk/
+
+Last-Updated: 2023-03-08 11:40:55+00:00
+Expires: 2023-06-08 11:40:55+00:00
```
Comment (samiwelthomasHO) [inline security.txt:9]: I would kick this further into the future, e.g. 2024-25?

**Review by samiwelthomasHO — CHANGES_REQUESTED** [review]

**Review by samiwelthomasHO — APPROVED** [review]

**Review by kayodeolajideHO — APPROVED** [review]

**Pull request merged**